jpeg/PSPBUILD

```diff
@@ -0,0 +1,32 @@
+# Maintainer: Wouter Wijsman <wwijsman@live.nl>
+pkgname=jpeg
+pkgver=8c
+pkgrel=1
+pkgdesc="a free library for JPEG image compression"
+arch=('mips')
+url="https://ijg.org/"
+license=('custom')
+depends=()
+makedepends=()
+optdepends=()
+source=("git+https://github.com/pspdev/psp-ports.git")
+sha256sums=('SKIP')
+
+prepare() {
+    cd psp-ports
+    git checkout f47bd37edf202284fc039106a5b27d9b7314bd76
```

Comment:
This can be done directly in source array:
`git+https://github.com/pspdev/psp-ports.git#commit=f47bd37edf202284fc039106a5b27d9b7314bd76`

Reply:
This is amazing, I should've known this earlier

Reply:
I've done this for all packages which use git now.